src/OfflineQuery.js

```diff
@@ -83,11 +83,11 @@ function equalObjectsGeneric(obj, compareTo, eqlFn) {
 /**
  * Converts human readable relative date string
  * ie: 'in 10 days' to a date relative to now
+ * copied from parse-server/src/Adapters/Storage/Mongo/MongoTransform.js
  *
  * @param {*} text
  * @param {*} now
- * @returns {status,info,result}
- * copied from parse-server/src/Adapters/Storage/Mongo/MongoTransform.js
+ * @returns {{status:string,info:string,result:Date}}
```

Comment:
The docs only say return type is `Object`, without any more details:

![image](https://user-images.githubusercontent.com/5673677/126822388-8e74e280-9e98-46c5-9a43-3696f8650967.png)

Is that what you intended? Or should we add a @typedef to document that output object in detail?

Reply:
I added a @typedef to complete the docs, is that ok?

![image](https://user-images.githubusercontent.com/5673677/126823987-4ed8f16f-8c14-47fe-9421-6fa451546966.png)